CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+
+## [2.0.3] - 2017-03-06
 ## Added
 - dashboards can now be created using the API and uploaded to Plotly. Use `import plotly.dashboard_objs` to be create a dashboard a `Dashboard` object. You can learn more about `Dashboard` objects by running `help(plotly.dashboard_objs.Dashboard)` and `help(plotly.plotly.plotly.dashboard_ops)` for uploading and retrieving dashboards from the cloud.
```

Comment:
@Kully 
- Capitalize 1st word of sentence (Dashboards).
- incomplete sentence: 
>Use `import plotly.dashboard_objs` to be create a dashboard a `Dashboard` object.